### DISCUSSION TITLE
why need Approval , push user

### DISCUSSION THREAD

**Discussion by Nishant9947** [ROM-builders/temporary#15312] [Q&A]
Sir , why i face regularly Approval in pr when i am push user
Without any successed build.
**Can u please fix it or give me approval permission.**

**Github** - https://github.com/Nishant9947
**Telegram** - https://t.me/Nishant9947

https://cirrus-ci.com/build/5621696250511360

![Screenshot_20221116-200504_Chrome](https://user-images.githubusercontent.com/82361039/202208896-24f37ba9-65a8-42d0-8a60-a776f71cdbb7.png)

**Comment by Apon77** [comment]
I have no idea why this happens

**Reply by Nishant9947** [reply]
So , sir can you please allow me to approve builds.
I am just maintaining Evolution X for X01AD.
I build monthly but cause of approval, i am unable to fix a13 bugs.

But Thank you & cirrus ci :) for this service
